source/dyaml/event.d

```diff
@@ -221,7 +221,7 @@ Event documentEndEvent(const Mark start, const Mark end, const bool explicit) pu
 ///          style    = Scalar style.
 Event scalarEvent(const Mark start, const Mark end, const Anchor anchor, const Tag tag,
                   const Tuple!(bool, bool) implicit, const string value,
-                  const ScalarStyle style = ScalarStyle.Invalid) @safe pure nothrow @nogc
+                  const ScalarStyle style = ScalarStyle.Invalid) @trusted pure nothrow @nogc
```

Comment:
Yeah, to be honest I don't really know if this patch is actually required - I will test with the latest DMD version to see if this actually fixes any current issue and - if not - drop it entirely.
The previous code looks fine to me ^^